analyzer/pkg/rpm/marinerdistroless.go

```diff
@@ -0,0 +1,114 @@
+package rpm
+
+import (
+	"bufio"
+	"context"
+	"os"
+	"strings"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/types"
+	"github.com/aquasecurity/fanal/utils"
+	"github.com/aquasecurity/go-dep-parser/pkg/io"
+	"golang.org/x/xerrors"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&marinerDistrolessPkgAnalyzer{})
+}
+
+const versionMarinerDistroless = 1
+
+var (
+	requiredMarinerDistrolessFiles = []string{"var/lib/rpmmanifest/container-manifest-2"}
+)
+
+type marinerDistrolessPkgAnalyzer struct{}
+
+func (a marinerDistrolessPkgAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+	pkgs, err := a.parseMarinerDistrolessManifest(input.Content)
+	if err != nil {
+		return nil, xerrors.Errorf("failed to parse mariner distroless 'container-manifest-2' file: %w", err)
+	}
+	return &analyzer.AnalysisResult{
+		PackageInfos: []types.PackageInfo{
+			{
+				FilePath: input.FilePath,
+				Packages: pkgs,
+			},
+		},
+	}, nil
+}
+
+func (a marinerDistrolessPkgAnalyzer) parseMarinerDistrolessManifest(r io.ReadSeekerAt) ([]types.Package, error) {
+	var pkgs []types.Package
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := scanner.Text()
+		var name, ver, rel, sourceRpm, arch string
+		// %{NAME}\t%{VERSION}-%{RELEASE}\t%{INSTALLTIME}\t%{BUILDTIME}\t%{VENDOR}\t(none)\t%{SIZE}\t%{ARCH}\t%{EPOCHNUM}\t%{SOURCERPM}
+		if s := strings.Split(line, "\t"); len(s) == 10 {
```

Comment:
I merged this, but after i found, that we can't use `strings.Fields` because `VENDOR` field can contain spaces(e.g. `Microsoft Corporation`.
Reverted `strings.Split` function.